src/SonarScanner.MSBuild.PreProcessor/Resources.resx

```diff
@@ -282,6 +282,9 @@ Use '/?' or '/h' to see the help message.</value>
   <data name="WARN_ExistingGlobalTargets" xml:space="preserve">
     <value>This version of the SonarScanner for MSBuild automatically deploys {0}, however a copy has been found in {1}. Please remove it if it is not intentional.</value>
   </data>
+  <data name="WARN_SonarQubeNotSupported" xml:space="preserve">
```

Comment:
To match the actuall error text.
```suggestion
  <data name="WARN_SonarQubeDeprecated" xml:space="preserve">
```

Reply:
I don't see this in commit history (yet)

Reply:
Yep forgot to commit it, sorry :)